internal/database/org_invitations.go

```diff
@@ -133,6 +134,21 @@ func (s *orgInvitationStore) Create(ctx context.Context, orgID, senderUserID, re
 	return t, nil
 }
 
+// GetPendingByOrgID retrieves the pending invitations for the given organization.
+//
+// 🚨 SECURITY: The caller must ensure that the actor is permitted to view this org invitation.
+func (s *orgInvitationStore) GetPendingByOrgID(ctx context.Context, orgID int32) ([]*OrgInvitation, error) {
+	results, err := s.list(ctx, []*sqlf.Query{
+		sqlf.Sprintf("org_id=%d AND responded_at IS NULL AND revoked_at IS NULL", orgID),
```

Comment:
We need to exclude expired invitations as well
```suggestion
		sqlf.Sprintf("org_id=%d AND responded_at IS NULL AND revoked_at IS NULL AND expires_at > now()", orgID),
```